baseplate/server/thrift.py

```diff
@@ -7,10 +7,15 @@
 from typing import Tuple
 from typing import Union
 
+import wrapt
```

Comment:
Note the new dependency.  This replaces 200 lines of boilerplate.  Our choice whether we carry this to production, but having an ObjectProxy around really helps a lot of framework use cases.  For example, I'm not sure how we currently send statsd for every thrift method, but an ObjectProxy would do it nicely.